fedimint-testing/src/bin/fixtures.rs

```diff
@@ -151,11 +151,12 @@ async fn await_fedimint_block_sync() -> anyhow::Result<()> {
 }
 
 async fn run_bitcoind() -> anyhow::Result<()> {
-    let btc_dir = env::var("FM_BTC_DIR")?;
-
-    // spawn bitcoind
+    let current_dir = env::current_dir().unwrap();
```

Comment:
should we assume current_dir is same as root of source code?

Reply:
I think `env::current_dir()` will return the path to `fixture` binary, and not the path to source code. It makes more sense to export `SRC_DIR` in `build.sh` and read that env variable here instead.